config/sitemap.rb

```diff
@@ -0,0 +1,39 @@
+require 'rubygems'
+require 'sitemap_generator'
+
+SitemapGenerator::Sitemap.default_host = 'https://pulsearch.princeton.edu'
```

Comment:
It would be better to get this from the Rails config instead of hard-coding `pulsearch.princeton.edu` — since that's not the staging URL, and won't be the production URL for very much longer.

Reply:
I have addressed this, but this PR is now blocked until princeton_ansible/issues/114 is resolved.